src/components/Homepage/Feed.js

```diff
@@ -0,0 +1,26 @@
+function Topics() {
+  return (
+    <article className="py-4 px-[25%]">
```

Comment:
This should not be an article, it should be a div instead. 
https://bitsofco.de/sectioning-content-in-html5/#:~:text=So%20which%20should%20you%20use,then%20use%20an%20.

Reply:
Okay, I changed the article to div. Thanks for clarifying.